WebRoot/skins/_base/base4/skin.css

```diff
@@ -1066,3 +1070,11 @@ div.ZmHtmlEditor .TxtAreaDrag {
 .ZmConvDoublePaneView .Row-selected .ZmMsgListDate {
     background-color: @ListItemBgColor-selected@ !important;
 }
+
+.newTaskBanner {
```

Comment:
.newTaskBanner defines its styles from property 'TaskListViewNewTaskBannerWrapper' defined in  properties file.
Now, property TaskListViewNewTaskBannerWrapper has again two property contained in it namely Text-hint and BottomSeparator which are skin/color dependent , so placed here.